src/interfaces/earnings.spec.ts

```diff
@@ -270,8 +270,69 @@ describe("EarningsInterface", () => {
     });
   });
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-function
-  describe.skip("assetHistoricEarnings", () => {});
+  describe("assetHistoricEarnings", () => {
```

Comment:
Added only the happy path, unsure if there's anything else you'd like to test. Seems pretty straightforward.